### ISSUE TITLE
WYSIWYG editor example (tiptap third party plugin integration): lists not working

### ISSUE THREAD

**Issue by RSchmitzHH** [htmlstreamofficial/preline#342]
Hi there! 

I really love that you integrated tiptap's WYSIWYG editor! However, in your [example](https://preline.co/docs/text-editor.html) in the docs, the lists (incl. ordered lists, indent) seem not to work so far. Tested with Chrome and Safari. 

Cheers,
Rüdiger

**Comment by olegpix** [comment]
> Hi there!
> 
> I really love that you integrated tiptap's WYSIWYG editor! However, in your [example](https://preline.co/docs/text-editor.html) in the docs, the lists (incl. ordered lists, indent) seem not to work so far. Tested with Chrome and Safari.
> 
> Cheers, Rüdiger

Hi!
Thanks for the notice. You're right, the issue is really exists. The problem could be solved by mapping different versions of `prosemirror-model` to the one.
In our case the following code helped:
```
<script type="importmap">
    {
      "imports": {
        "https://esm.sh/v135/prosemirror-model@1.19.3/es2022/prosemirror-model.mjs": "https://esm.sh/v135/prosemirror-model@1.20.0/es2022/prosemirror-model.mjs",
        "https://esm.sh/v135/prosemirror-model@1.19.4/es2022/prosemirror-model.mjs": "https://esm.sh/v135/prosemirror-model@1.20.0/es2022/prosemirror-model.mjs"
      }
    }
  </script>
  ```
  The issue appears because of different versions of the `prosemirror-model` in some imports here
  ```
  import { Editor } from 'https://esm.sh/@tiptap/core';
    import Placeholder from 'https://esm.sh/@tiptap/extension-placeholder';
    import StarterKit from 'https://esm.sh/@tiptap/starter-kit';
    import Paragraph from 'https://esm.sh/@tiptap/extension-paragraph';
    import Bold from 'https://esm.sh/@tiptap/extension-bold';
    import Underline from 'https://esm.sh/@tiptap/extension-underline';
    import Link from 'https://esm.sh/@tiptap/extension-link';
    import BulletList from 'https://esm.sh/@tiptap/extension-bullet-list';
    import OrderedList from 'https://esm.sh/@tiptap/extension-ordered-list';
    import ListItem from 'https://esm.sh/@tiptap/extension-list-item';
    import Blockquote from 'https://esm.sh/@tiptap/extension-blockquote';
  ```

**Comment by jahaganiev** [comment]
Hey @RSchmitzHH - the issue has been fixed in 2.3.0 update.
